proms-resources/feature/src/main/features/feature.json

```
@@ -24,13 +24,25 @@
     {
       "id":"${project.groupId}:proms-frontend:${project.version}",
       "start-order":"26"
+    },
+    {
+      "id":"${project.groupId}:proms-backend:${project.version}",
+      "start-order":"26"
     }
   ],
   "configurations":{
     "org.apache.sling.jcr.repoinit.RepositoryInitializer~proms":{
       "service.ranking:Integer":300,
       "scripts": [
-        "create user patient \n set ACL for patient \n     allow jcr:read on /Questionnaires,/SubjectTypes \n     deny jcr:all on /Questionnaires restriction(rep:itemNames,audit_intro,audit_results,eq5d_results,gad7_intro,gad7_results,phq9_intro,phq9_results) \n     allow jcr:read,jcr:write,jcr:nodeTypeManagement,jcr:versionManagement on /Forms restriction(cards:sessionSubject) \n     allow jcr:read on /Subjects restriction(cards:sessionSubject) \n     deny jcr:modifyProperties on /Forms restriction(rep:itemNames,subject,questionnaire) \n end"
+        "create user patient \n set ACL for patient \n     allow jcr:read on /Questionnaires,/SubjectTypes \n     deny jcr:all on /Questionnaires restriction(rep:itemNames,audit_intro,audit_results,eq5d_results,gad7_intro,gad7_results,phq9_intro,phq9_results) \n     allow jcr:read,jcr:write,jcr:nodeTypeManagement,jcr:versionManagement on /Forms restriction(cards:sessionSubject) \n     allow jcr:read on /Subjects restriction(cards:sessionSubject) \n     deny jcr:modifyProperties on /Forms restriction(rep:itemNames,subject,questionnaire) \n end"]
+    },
+    "org.apache.sling.jcr.repoinit.RepositoryInitializer~proms_backend":{
+      "service.ranking:Integer":200,
+      "scripts":["create service user proms-visit-backend \n set ACL on /Forms,/Questionnaires,/Subjects,/SubjectTypes \n   allow jcr:read for proms-visit-backend \n end \n set ACL on /Forms \n   allow jcr:read,rep:write,jcr:versionManagement for proms-visit-backend \n end \n"]
+    },
+    "org.apache.sling.serviceusermapping.impl.ServiceUserMapperImpl.amended~":{
```

This needs a specific name:

```suggestion
    "org.apache.sling.serviceusermapping.impl.ServiceUserMapperImpl.amended~proms":{
```